packages/core/src/ceramic.ts

```diff
@@ -224,15 +224,17 @@ export class Ceramic implements CeramicApi {
       this._streamHandlers
     )
     const pinStore = modules.pinStoreFactory.createPinStore()
+    const localIndex = new LocalIndexApi(modules.indexing, this.repository, this._logger)
     this.repository.setDeps({
       dispatcher: this.dispatcher,
       pinStore: pinStore,
       context: this.context,
       handlers: this._streamHandlers,
       anchorService: modules.anchorService,
       conflictResolution: conflictResolution,
+      indexing: localIndex,
     })
-    this._index = new LocalIndexApi(modules.indexing, this.repository, this._logger)
+    this._index = localIndex
```

Comment:
are you going to get rid of this in a follow-up PR?  If so can you add a TODO with the ticket number?

Reply:
correct, adding TODO right now.